docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.7"
+
+services:
+  dashi:
+    container_name: dashi
+    build: 
+      context: .
+      dockerfile: Dockerfile
+    restart: always
+    ports:
+      - '5000:5000'
+    volumes:
+      # location of gsi-qc-etl cache files
+      - type: bind
+        source: ./cache_files
```

Comment:
It would be nice if this could be not hardcoded; the location of this is stored in the `$GSI_QC_ETL_ROOT_DIRECTORY` variable present on the host machine. This is not a need-to-have though. Everything else works as-is.

Reply:
Changed it to point to `$HOME/qcetl`.